src/sentry/grouping/grouping_info.py

```diff
@@ -14,9 +14,9 @@
 
 
 def get_grouping_info(
-    config_name: str | None, project: Project, event_id: str
+    config_name: str | None, project: Project, event_id: str, event: Event | None = None
```

Comment:
nit: shouldn't `event_id` be optional then? just get it from the Event when it's None.